docs/index.rst

```diff
@@ -10,18 +10,18 @@ SHARE harvests metadata nightly from 100+ repositories, transforms that metadata
 
 The technical side of SHARE has many pieces that you can interact with:
 
-- API Endpoints for accessing the raw and transformed metadata
-    + https://share.osf.io/api
 - A search endpoint powered by elasticsearch that indexes the transformed data allowing:
    - Thorough search of creative works:
         + Creative works: `</api/v2/search/creativeworks/_search_>`_
         + more info on the `elasticsearch docs`_ page
    - Data aggregations across fields
-- A python library called ``sharepa`` for SHARE Parsing and Analysis
 - An `Ember application`_ using the SHARE API for:
    - Discovering new Projects
    - Curation
    - Corrections and Updates
+- A python library called ``sharepa`` for SHARE Parsing and Analysis
```

Comment:
Nope, but that is definitely a different ticket. There is a PR on sharepa for a needed update. (I did not add a link to sharepa from the docs it already existed in multiple places)